layers/+lang/markdown/config.el

```diff
@@ -37,3 +37,6 @@ generate mmm classes.")
 
 (defvar markdown--key-bindings-modes '(markdown-mode gfm-mode)
   "Modes using markdown key bindings.")
+
+(defvar markdown-fmt-tool 'prettier
```

Comment:
AFAIK, you need to install `prettier` using `npm`. If so, it is a bad default as not every `markdown` user uses `npm`. And what happens if you don't have `prettier` in your `PATH`? Do you see errors when you edit/save markdown buffers?